geomesa-accumulo/geomesa-accumulo-tools/src/test/scala/org/locationtech/geomesa/accumulo/tools/authentication/AuthenticationTest.scala

```diff
@@ -1,84 +0,0 @@
-/***********************************************************************
- * Crown Copyright (c) 2017-2020 Dstl
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Apache License, Version 2.0
- * which accompanies this distribution and is available at
- * http://www.opensource.org/licenses/apache2.0.php.
- ***********************************************************************/
-
-package org.locationtech.geomesa.accumulo.tools.authentication
-
-import java.io.File
-
-import com.typesafe.config.{ConfigFactory, ConfigRenderOptions}
-import org.junit.runner.RunWith
-import org.locationtech.geomesa.accumulo.tools.{AccumuloDataStoreCommand, AccumuloRunner}
-import org.locationtech.geomesa.utils.collection.SelfClosingIterator
-import org.locationtech.geomesa.utils.geotools.Conversions._
-import org.specs2.mutable.Specification
-import org.specs2.runner.JUnitRunner
-
-@RunWith(classOf[JUnitRunner])
-class AuthenticationTest extends Specification {
-
-  sequential
-
-  "GeoMesa Accumulo Commands" should {
-
-    // These tests all invoke an ingest command using a mock instance
-    // The authentication parameters aren't actually used by the mock instance, but these tests serve to test parameter
-    // validation in AccumuloRunner and associated classes.
-
-    val conf = ConfigFactory.load("examples/example1-csv.conf")
-    val sft = conf.root().render(ConfigRenderOptions.concise())
-    val converter = conf.root().render(ConfigRenderOptions.concise())
-    val dataFile = new File(this.getClass.getClassLoader.getResource("examples/example1.csv").getFile)
-
-    // Only test ingest, other commands use the same Accumulo Parameters anyway
-    val cmd = Array("ingest")
-
-    "fail without user" >> {
-      val authArgs = Array("")
-      val args = cmd ++ authArgs ++ Array("--instance", "instance", "--zookeepers", "zoo", "--mock", "--catalog", "z", "--converter", converter, "-s", sft, dataFile.getPath)
-      val command = AccumuloRunner.parseCommand(args).asInstanceOf[AccumuloDataStoreCommand] must throwA[com.beust.jcommander.ParameterException]
-      ok
-    }
-
-    // With just user parameter, the user is prompted to enter a password at the command line  which we can't test here
-
-    "work with user and password" >> {
```

Comment:
Do we still have these test cases around somewhere?

Reply:
yeah this test was badly named... i moved it to AccumuloDataStoreFactoryTest

Reply:
I suppose this was functioning as a mini-integration test that shows that the command line tooling passes things through.  Meh, do we want something like this?

Reply:
oh, sorry the last comment was wrong, i was thinking of a different test case. these tests were consolidated into IngestCommandTest and/or AccumuloExportCommandTest. they're still around though